src/components/course/CoursePage.jsx

```diff
@@ -102,7 +102,18 @@ const CoursePage = () => {
   } = useAllCourseData({ courseService, activeCatalogs });
   const isEMETRedemptionEnabled = getConfig().FEATURE_ENABLE_EMET_REDEMPTION || hasFeatureFlagEnabled('ENABLE_EMET_REDEMPTION');
 
-  const validCourseRunKeys = getAvailableCourseRunKeysFromCourseData(courseData);
+  let anyPolicyHasLateRedemptionEnabled = false;
+  if (redeemableLearnerCreditPolicies?.redeemablePolicies) {
+    anyPolicyHasLateRedemptionEnabled = redeemableLearnerCreditPolicies.redeemablePolicies.some((policy) => (
+      // is_late_redemption_enabled=True on the serialized policy represents the fact that late redemption has been
+      // temporarily enabled by an operator for the policy. It will toggle itself back to False after a finite period
+      // of time.
+      policy.isLateRedemptionEnabled
+    ));
```

Comment:
Might be able to keep `anyPolicyHasLateRedemptionEnabled ` defined as a const and have the `.some` iteration where the variable is defined. If `redeemableLearnerCreditPolicies` is undefined/not present, the `!!` will cast the `undefined` resulting from `redeemableLearnerCreditPolicies?.redeemablePolicies` as `false`.

```jsx
  const anyPolicyHasLateRedemptionEnabled = !!redeemableLearnerCreditPolicies?.redeemablePolicies.some((policy) => (
    // is_late_redemption_enabled=True on the serialized policy represents the fact that late redemption has been
    // temporarily enabled by an operator for the policy. It will toggle itself back to False after a finite period
    // of time.
    policy.isLateRedemptionEnabled
  );
```